<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jogo da Vida 3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblRows">
            <property name="text">
             <string>Linhas:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRows">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="keyboardTracking">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblCols">
            <property name="lineWidth">
             <number>-7</number>
            </property>
            <property name="text">
             <string>Colunas:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxCols">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblSlices">
            <property name="text">
             <string>Fatias:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxSlices">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblIteration">
          <property name="text">
           <string>Iteração: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zerarButton">
          <property name="text">
           <string>&amp;Zerar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlay">
          <property name="text">
           <string>&amp;Play -&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="iterarButton">
          <property name="text">
           <string>&amp;Iterar -&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="text">
           <string>&amp;Sair</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
